packages/core/src/Stores/ui-store.js

```diff
@@ -932,6 +937,14 @@ export default class UIStore extends BaseStore {
         this.sub_section_index = index;
     }
 
+    setIsVerificationModalVisible(value) {
+        this.is_verification_modal_visible = value;
+    }
+
+    setIsVerificationSubmitted(value) {
+        this.is_verification_submitted = value;
+    }
```

Comment:
Do we need this extra flag?

Reply:
Yes, this is to make the submitted-modal pop up visible after user submit their document